app/views/admin/submissions/_form.html.erb

```diff
@@ -22,7 +22,7 @@
               </label>
               <div class='col-sm-10' id='user_selections_form'>
                 <%= f.text_field :user, class: 'form-control', id: 'user_search' %>
-                <%= f.hidden_field :user_id %>
+                <%= f.hidden_field :user_id, :value => @submission&.user&.gravity_user_id %>
```

Comment:
Little context here: previously, when a user first loaded the form, the `value` in this field was the Convection user id, not the Gravity user id. After searching for a user, it would change to be the Gravity user id.

This was never a problem because we didn't permit the `user_id` param, so you could not update the user associated with a submission after it was created.

Because we're now permitting that param and we want to use it to update users associated with submissions, we needed to make this consistent! So now it's always the gravity ID of the user associated with the submission, and we find the convection user based on that ID.